src/components/UserComboBox/UserComboBox.tsx

```diff
@@ -67,36 +67,38 @@ export const UserComboBox = ({ value, onChange, placeholder, blank }: UserComboB
                         </div>
                     )}
                 />
-                <DropdownPanel placement="bottom-start">
-                    <Input
-                        placeholder={placeholder}
-                        outline
-                        autoFocus
-                        onChange={(e: ChangeEvent<HTMLInputElement>) => {
-                            setSearch(e.currentTarget.value);
-                            suggestionsVisibility.setTrue();
-                        }}
-                        iconLeft={<IconSearchOutline size="s" />}
-                    />
-                    <ListView>
-                        {userListQuery.data?.users?.map((user) => (
-                            <ListViewItem
-                                key={user.id}
-                                renderItem={({ active, hovered, ...props }) => (
-                                    <MenuItem hovered={active || hovered}>
-                                        <HarmonyUser
-                                            className={s.ListItem}
-                                            onClick={() => onUserChange(user)}
-                                            name={user.name}
-                                            email={user.email}
-                                            {...props}
-                                        />
-                                    </MenuItem>
-                                )}
-                            />
-                        ))}
-                    </ListView>
-                </DropdownPanel>
+                {nullable(suggestionsVisibility.value, () => (
+                    <DropdownPanel placement="bottom-start">
```

Comment:
Maybe we can pass `visible` prop to dropdown panel

Reply:
will fix that in bricks